website/content/labs/advanced-git.md

```diff
@@ -495,106 +495,88 @@ includes a line telling us which two commits were merged:
 
-Now, from inside the repository, you will create a branch to make your
-change to `AUTHORS.md`. But don't do it yet! Before you do this, let's 
+    https://github.com/cmsc22000-labs/2020-names
+    
+Go ahead and clone this repository. This repository contains a single ``README.md`` file
+with a list of names. You will be creating a branch to add your name to 
+`README.md`. But don't do it yet! Before you do this, let's 
 take a minute to talk about naming conventions.
 
-If everyone named their branch something like `authors` or `add-authors`, we'd
-have 5 branches with the same name. In large collaborative projects, it's common
+If everyone named their branch something like `update-readme` or `add-name`, we'd
+have lots of branches with the same name. In large collaborative projects, it's common
 to prefix branch names with your own first name, to avoid collisions like this.
-We will adopt this convention in this class, except we will use CNetIDs instead
-of names. If your CNetID is `jdoe`, you should
-name your branch `jdoe/add-to-authors` or `jdoe/authors` or similar. It doesn't
-really matter what the name of the branch is, so long as it starts with `jdoe/`.
+We will adopt a similar convention in this class, except we will use feature names. So,
+if you are working on adding sound support to chiventure, you branches would be called
+``sound/fix-alsa-bug``, ``sound/add-midi-support``, etc.
 
-So, go ahead and create a new branch and check it out:
-
-```
-$ git branch CNETID/add-to-authors
+For this sample repo, we'll just use your CNetID. So, if your CNetID is `jdoe`, you should
+name your branch `jdoe/add-to-names` or `jdoe/update-readme` or similar. It doesn't
+really matter what the name of the branch is, so long as it starts with `jdoe/`.
 
-$ git checkout CNETID/add-to-authors
-```
+Using the commands we described earlier in this lab, you should now do the following:
 
-Then, edit the AUTHORS.md file to add your name. Make a commit and push the
-new branch:
+1. Create and check out a branch called ``CNETID/add-to-names`` (or similar, as long as
+  it starts with your CNetID)
+2. Edit the `README.md` file to add your name, and create a commit with that change.
+3. Push your branch to GitHub. Git may helpfully provide you with a URL to create
+   a pull request; ignore this for now, although you're welcome to use that direct
+   URL in the future.
 
-```
-$ git push origin CNETID/add-to-authors
-```
+Now, go to the repository on GitHub (https://github.com/cmsc22000-labs/2020-names). It
+should actually show that you've pushed a branch recently, and there will be a green 
+"Compare & pull request" button. Click on this button to create a pull request.
+If you don't see a "Compare & pull request" button, you can also click on the 
+"New pull request" button that appears next to the pull-down list of branches; you
+will then have to manually specify the branch you're merging before you can create
+the pull request.
```

Comment:
It would be helpful to mention that the base branch should be `master` and the compare branch would be `CNETID/add-to-names`

Reply:
Fixed.